CONTRIBUTING.md

```diff
@@ -0,0 +1,86 @@
+# 🤝***First time contributing? We will help you out.*** 👍
+
+![GitHub custom open for collaboration](https://img.shields.io/badge/Open%20For-Collaboration-brightgreen?style=for-the-badge)
+
+Refer to the following articles on the basics of Git and Github. You can also contact the Project Mentors, in case you are stuck:
+
+
+- [Getting started with Git and GitHub](https://docs.github.com/en/free-pro-team@latest/github/getting-started-with-github)
+- [Forking a Repo](https://help.github.com/en/github/getting-started-with-github/fork-a-repo)
+- [Cloning a Repo](https://help.github.com/en/desktop/contributing-to-projects/creating-a-pull-request)
```

Comment:
```suggestion
- [Cloning a Repo](https://docs.github.com/en/github/creating-cloning-and-archiving-repositories/cloning-a-repository)
```
This is a better article for cloning a repo. Previous one directed to wrong article(creating a pull request)